### PR TITLE
Makin' it work for OSX

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
R? @cadecairos

File: README.md [hunk 1]
```diff
@@ -31,27 +32,30 @@ The second `docker-compose up -d` we run in `/services` launches the Webmaker se
 I've put the Dockerfiles for these in the [services-dockerfiles](/services-dockerfiles) folder of this repo. There's also a `.dockerignore` file there that's applicable to each project.
 
 ## What's running where, and what are my creds
-* Postgres is listening on localhost:5432
+
+After your Docker containers are all running, you'll need the IP of the Docker VM. To get this, run `docker-machine ip default`.
```
Comment (cadecairos) [inline README.md]: This is only true on Mac and Windows, could we change this to reflect that?

Comment (gvn) [inline README.md]: Ah yeah, I realized that the other day. I'll make some tweaks...

**Comment by gvn** [comment]
Ok, updated the README. Weirdly, I'm having problems getting Postgres to run again.

``` bash
pg_1      | The files belonging to this database system will be owned by user "postgres".
pg_1      | This user must also own the server process.
pg_1      |
pg_1      | The database cluster will be initialized with locale "en_US.utf8".
pg_1      | The default database encoding has accordingly been set to "UTF8".
pg_1      | The default text search configuration will be set to "english".
pg_1      |
pg_1      | Data page checksums are disabled.
pg_1      |
pg_1      | fixing permissions on existing directory /var/lib/postgresql/data/pg_data ... ok
pg_1      | creating subdirectories ... initdb: could not create directory "/var/lib/postgresql/data/pg_data/global": Permission denied
pg_1      | initdb: removing contents of data directory "/var/lib/postgresql/data/pg_data"

dataservices_pg_1 exited with code 1
```

**Comment by cadecairos** [comment]
darn! Volumes are a PITA

**Comment by cadecairos** [comment]
@gvn I think I might try this to solve our volume mounting woes: http://container42.com/2013/12/16/persistent-volumes-with-docker-container-as-volume-pattern/

TL;DR; A separate container that exposes a mountable volume. It should be cross platform friendly too :+1:
